src/components/ChooseToken.vue

```diff
@@ -202,6 +204,9 @@ export default Vue.extend({
     displayTokenBalance(): boolean {
       return !this.onlyMintTokens && (this.tokensType === "L1-Tokens" || this.tokensType === "L2-Tokens");
     },
+    provider(): RestProvider {
```

Comment:
what's the purpose of this?

Reply:
I need the provider to perform [this](https://github.com/rsksmart/rif-rollup-wallet/pull/143/files#diff-be1ce0dcd6f3e04c3beb212a7c84e826c72060287811b74232a61349ff530628R229) call.